src/cats/util.cljc

```diff
@@ -27,6 +27,12 @@
   "A collection of utils that used around the library."
   (:require [cats.protocols]))
 
+(defmacro flattenl-1 [coll]
```

Comment:
It seems to me this should be a (private) function, not a macro, and live in `builtin.cljc`.